p5js/ImageClassification/sketch.js

```diff
@@ -5,12 +5,11 @@
 
 /* ===
 ML5 Example
-ImageNet_Simple
 Simple Image Classification using p5.js
 === */
 
-// Initialize the ImageNet method with the MobileNet model.
-const classifier = new ml5.ImageClassifier('MobileNet');
+// Initialize the Image Classifier method
+const classifier = new ml5.ImageClassifier();
```

Comment:
Is "MobileNet" the default but the Image classifier could still load other models as well? I think I like the idea of requiring a reference to the pre-trained model to increase transparency. And this also opens up the possibility of creating an "empty" Image Classifier that has to be trained from scratch?

Reply:
It is the default because Squeezenet does not currently work in tf.js (See: https://github.com/tensorflow/tfjs-core/tree/master/models)
I also liked specifying the model, we can bring it back!